apps/src/tests/index.ts

```diff
@@ -0,0 +1,109 @@
+// @ts-nocheck
+
+export { default as Test42 } from './Test42';
```

Comment:
```suggestion
export { Test42 } from './Test42';
```

Shouldn't it be enough?

Reply:
Yeah, I've tested it, should be enough.

Reply:
Oh, you're right, we export each test as `default` from each test module & we need to reexport it in such way. Okey, that's fine then. 👍🏻